nms/app/packages/magmalte/server/network/routes.js

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow
+ * @format
+ */
+
+import type {ExpressResponse} from 'express';
+import type {FBCNMSRequest} from '@fbcnms/auth/access';
+import type {
+  network_cellular_configs,
+  network_dns_config,
+  tier,
+} from '@fbcnms/magma-api';
+
+import asyncHandler from '@fbcnms/util/asyncHandler';
+import express from 'express';
+
+import MagmaV1API from '../magma';
+import {AccessRoles} from '@fbcnms/auth/roles';
+import {CWF, FEG, FEG_LTE, LTE, SYMPHONY, XWFM} from '@fbcnms/types/network';
+import {access} from '@fbcnms/auth/access';
+import {difference} from 'lodash';
+
+const logger = require('@fbcnms/logging').getLogger(module);
+
+const router: express.Router<FBCNMSRequest, ExpressResponse> = express.Router();
+
+const DEFAULT_CELLULAR_CONFIG: network_cellular_configs = {
+  epc: {
+    cloud_subscriberdb_enabled: false,
+    default_rule_id: 'default_rule_1',
+    lte_auth_amf: 'gAA=',
+    lte_auth_op: 'EREREREREREREREREREREQ==',
+    mcc: '001',
+    mnc: '01',
+    network_services: ['policy_enforcement'],
+    relay_enabled: false,
+    sub_profiles: {},
+    tac: 1,
+  },
+  feg_network_id: '',
+  ran: {
+    bandwidth_mhz: 20,
+    // TODO: Add option in UI for either fdd or tdd
+    // plus config values
+    // fdd_config: {
+    //   earfcndl: 44590,
+    //   earfcnul: 18000,
+    // },
+    tdd_config: {
+      earfcndl: 44590,
+      special_subframe_pattern: 7,
+      subframe_assignment: 2,
+    },
+  },
+};
+
+// A placeholder due to bug in serialization
+const NETWORK_FEATURES = {
+  features: {
+    features: {
+      placeholder: 'true',
+    },
+  },
+};
+
+const DEFAULT_DNS_CONFIG: network_dns_config = {
+  enable_caching: false,
+  local_ttl: 0,
+  records: [],
+};
+
+const DEFAULT_UPGRADE_TIER: tier = {
+  gateways: [],
+  id: 'default',
+  images: [],
+  name: 'Default Tier',
+  version: '0.0.0-0',
+};
+
+router.post(
+  '/create',
+  access(AccessRoles.SUPERUSER),
+  asyncHandler(async (req: FBCNMSRequest, res) => {
+    const {networkID, data} = req.body;
+    const {name, description} = data;
+    const commonField = {
+      name,
+      description,
+      id: networkID,
+      ...NETWORK_FEATURES,
+    };
+
+    let resp;
+    try {
+      if (data.networkType === LTE) {
```

Comment:
This feels like it would be better as a switch statement, but that's just personal preference.

Reply:
I agree. This was again borrowed code. I can clean this up in a different PR.